### PR TITLE
Refactors EE code

### PR DESCRIPTION
**Pull request by zomars**
To be merged with 
- calcom/console#25
- calcom/api#150
- calcom/website#181

## What does this PR do?

- Introduces `@calcom/features` for future code organization [RFC here](https://github.com/calcom/cal.com/discussions/3571)
- Gets rid of `apps/web/ee` in favor of `packages/features/ee`
- Now you import via `@calcom/features/ee` instead of `@ee/` alias
- Applies "per feature module" file structure for `@calcom/features /ee` (similar to our console project)
  - Ex. all `workflows` code is as colocated as possible.
  - Same for `web3`, `support`, `teams`, `api-keys`, `impersonation` and `payments` feature modules.
- Extract multiple ui components to `@calcom/ui` (to be finished in #3051)
- Gets rid of `@lib/prisma` in favor of `@calcom/prisma` (technical debt)
- Consolidates Providers in `apps/web/lib/app-providers.tsx`
- Consolidates all stripe code in `@calcom/app-store/stripepayments`
- Removes `@calcom/stripe`

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How should this be tested?

- [ ] Test workflows
- [ ] Test payments
- [ ] Test web3
- [ ] Test chat inputs
- [ ] Test api key generation

### CODE REVIEW

File: packages/ee/api-keys/components/ApiKeyListContainer.tsx [hunk 1]
```diff
@@ -34,10 +32,10 @@ function ApiKeyListContainer() {
...
       />
       <LicenseRequired>
-        <QueryCell
-          query={query}
-          customLoader={<SkeletonLoader />}
-          success={({ data }) => (
+        {(() => {
+          if (query.isLoading) return <SkeletonLoader />;
+          if (!data) return null;
+          return (
```
Comment (zomars) [inline packages/ee/api-keys/components/ApiKeyListContainer.tsx:38]: Tried to migrate `QueryCell` but it breaks Github CI. I can't figure out why

**Comment by zomars** [comment]
@hariombalhara any ideas on why tests are failing? Did I moved something? Not AFAIK
image

**Comment by hariombalhara** [comment]
@zomars Yeah this is unrelated. Fix here https://github.com/calcom/cal.com/pull/3560. End of month bug

File: packages/features/links/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Event types will become Links…
+
+…and this is where the feature will live.
```
Comment (zomars) [inline packages/features/links/README.md:3]: Thought @PeerRich @baileypumfleet ?

**Pull request merged**